lit/docs/auth/configuring/generic-saml.lit

```diff
@@ -0,0 +1,101 @@
+\title{Generic SAML auth}{generic-saml-auth}
+
+\use-plugin{concourse-docs}
+\omit-children-from-table-of-contents
+
+A Concourse server can authenticate against any valid SAML auth provider.
+
+\section{
+  \title{Authentication}{generic-saml-authentication}
+
+  First you'll need to create a client with your SAML provider.
+
+  The SAML Assertion Consumer Service (ACS) URL and the Audince URI (SP Entity ID)
+  must be the URL of your Concourse server with \code{/sky/issuer/callback} appended.
```

Comment:
Based on https://github.com/concourse/concourse/pull/5998#pullrequestreview-479785775, the `Audience URI (SP Entity ID)` doesn't need to be the callback URL. According to https://stackoverflow.com/a/24199710/9994448:

> It is *recommended* that the URI is a URL that contains the domain name of the entity.

...so it doesn't have to specifically have `/sky/issuer/callback` appended. Maybe you could make a note that this can be configured using `CONCOURSE_SAML_ENTITY_ISSUER`, but it defaults to the `/sky/issuer/callback` URL?

Also, small typo in `Audience`